<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SketchProperty</class>
 <widget class="QWidget" name="SketchProperty">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="TextLabelID">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LineEditID">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Geometry Properties</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="TextLabelPoint1">
          <property name="text">
           <string>Point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="LineEditPoint1"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Attributes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="TextLabelColor">
          <property name="text">
           <string>Color</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboBoxColor">
          <item>
           <property name="text">
            <string>BLACK</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BROWN</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>RED</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ORANGE</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>YELLOW</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="TextLabelType">
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboBoxType">
          <item>
           <property name="text">
            <string>Main</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Auxiliary</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="TextLabelStyle">
          <property name="text">
           <string>Style</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboBoxStyle">
          <item>
           <property name="text">
            <string>SOLID</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DASH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DOT</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DOTDASH</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="TextLabelWidth">
          <property name="text">
           <string>Width</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ComboBoxWidth">
          <item>
           <property name="text">
            <string>1.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8.0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9.0</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="PushButtonOK">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
